### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Julien-Molina**

### CODE REVIEW

File: src/communication/WebSocketWrapper.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {fromEvent as observableFromEvent,  Observable } from "rxjs";
```
Comment (maurelio1234) [inline src/communication/WebSocketWrapper.ts]: all that to make it testable?

Comment (maurelio1234) [reply src/communication/WebSocketWrapper.ts]: why not replacing the websocket by a subject (http://reactivex.io/rxjs/manual/overview.html#subject).
much more standard, and will make it easier to update later.

File: test/spec/WebSocketSubscriberSpec.ts [hunk 2]
```diff
@@ -140,16 +142,17 @@ describe("Test xcWebSocketSubscriber module", function () {
                             expect(jsonData.Header.StateMachineCode).toBe(Mock.stateMachineCode);
                             expect(jsonData.Header.ComponentCode).toBe(Mock.componentCode);
                             expect(jsonMessage.ReplyTopic).toBe(topic);
-                            expect(jsonMessage.CallerPrivateTopic[0]).toBe(Mock.privateTopic);
+                            // expect(jsonMessage.CallerPrivateTopic[0]).toBe(Mock.privateTopic);
```
Comment (maurelio1234) [inline test/spec/WebSocketSubscriberSpec.ts]: remove

File: src/communication/WebSocketWrapper.ts (hunk 1 above)
Comment (Julien-Molina) [reply src/communication/WebSocketWrapper.ts]: I don't think that it's possible to send messages with a Subject no ?
Here we also need the send function.

Comment (maurelio1234) [reply src/communication/WebSocketWrapper.ts]: With a next()?
I'm almost sure that's what we used in the py API

Comment (maurelio1234) [reply src/communication/WebSocketWrapper.ts]: https://github.com/xcomponent/ReactiveXComponent.py/blob/e8c661d76c4bc58cdd606e6d0762b51ec879bdf9/reactivexcomponent/reactivexcomponent/communication/xc_session.py

**Pull request merged**